website/pages/en/index.js

```diff
@@ -14,16 +14,19 @@ const siteConfig = require(process.cwd() + "/siteConfig.js");
 const translate = require("../../server/translate.js").translate;
 
 const pre = "```";
-const typeAnnotationExample= `${pre}hack
+const asyncExample= `${pre}hack
 <?hh
```

Comment:
remove - discouraged in new public code

Reply:
can you clarify what/which to remove and what's discouraged? thanks!

Reply:
the `<?hh` line is only needed in `.php` files, and is invalid in `.hack` and `.hackpartial` files; while FB still mostly uses `.php` files, `.hack` are preferred externally